🐛 Display Related Tables for Nodes without Relations (#797)
liam-hq/liam: .changeset/violet-dolphins-reflect.md

---
"@liam-hq/erd-core": patch
---

> 🐛 When nodes without a relation are present, set the hidden property to false for the non related group node.

Reviewer: Please describe the value gained by the user, not the implementation details.
example:

```suggestion
🐛 When nodes without relationships are present, display only that node in RelatedTables.
```

.changeset/late-swans-beam.md seems good to delete.
